.github/actions_scripts/get_release_vars.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
 
-VERSION=$(cat ${1}/Jamulus.pro | grep -oP 'VERSION = \K\w[^\s\\]*')
-echo "::set-output name=JAMULUS_VERSION::${VERSION}"
-# get the tag which pushed this
-echo "::set-output name=PUSH_TAG::${GITHUB_REF#refs/*/}"
+set -e 
+
+# get version from project-file
+JAMULUS_VERSION=$(cat ${1}/Jamulus.pro | grep -oP 'VERSION = \K\w[^\s\\]*')
+
+# get the tag/branch-name which pushed this
+PUSHED_NAME=${GITHUB_REF#refs/*/}
+
+# calculate various variables
+python3 ${1}/.github/actions_scripts/analyse_git_reference.py ${GITHUB_REF} ${PUSHED_NAME} ${JAMULUS_VERSION} 
+
 perl ${1}/.github/actions_scripts/getChangelog.pl ${1}/ChangeLog ${VERSION} > ${1}/autoLatestChangelog.md
```

Comment:
You will have to rename ${VERSION} to ${JAMULUS_VERSION} if you also renamed the variable above. 

Also Codacy complained about missing quotes. We might need to add them around the variables/file paths.

Reply:
thanks.
Where can i see these complaints when I develop something?

Reply:
You will not see them. I setup codacy on my repo only. You can just enable it if you create a codacy account (just google for it).

Reply:
Sorry guys, I accidentally used the PR branch for development. I will open a new request soon.